docs/guide/installation.md

```diff
@@ -64,7 +64,7 @@ If you are using `enableStrictParsing` URL manager option, add the following to
 ### Extra configuration for logging and profiling
 
 Logging and profiling are simple but powerful tools that may help you to understand the execution flow of both the
-framework and the application. These tools are useful for development and production environments alike.
+framework and the application. These tools are useful for development and production environments NOT alike.
```

Comment:
@samdark maybe I misunderstood, but following text recommend [log only significantly important messages **manually**](https://github.com/yiisoft/yii2-debug/pull/24/files#diff-177d229ddd0c5566821ce34da4196595R69) in production environment, and [use debugger features](https://github.com/yiisoft/yii2-debug/pull/24/files#diff-177d229ddd0c5566821ce34da4196595R72) in development environment.